File: C/Atividade/T3/2.c
#include <stdio.h>
#include <stdlib.h>

int main(){
  double har = 0.0, i, num;

  printf("Digite um inteiro para a série harmônica: ");
  scanf("%lf", &num);

  for (i=0; i < num; i++){
    har = har + (1/(i+1));
  }
  printf("%.16lf", har);
  return 0;
}

File: C/Atividade/T3/2.py
har: float = 0.0
num = 0
for c in range(201):
  har: float = 0.0
  for i in range(c):
    har += 1/(i+1)
  num=har*100+num
print(f"Lvl.{c} {har*100+num:.0f}")